backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -18,11 +26,83 @@
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = UserResponse.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"id\": \"a6fd6d95-0a60-43ff-961f-2b9b2ff72f95\","
+                                + " \"email\": \"username@gmail.com\","
+                                + "  \"name\": \"Name\","
+                                + "  \"surname\": \"Surname\","
+                                + "  \"phoneNumber\": \"38166575757\","
+                                + "  \"address\": \"Local address\"}")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "400",
+            description = "Input fields are not in valid format",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"password format not valid\","
+                                + " \"name cannot be null or empty\","
+                                + " \"surname cannot be null or empty\","
+                                + " \"email format is not valid\","
+                                + " \"phoneNumber format not valid\"],"
+                                + " \"status\": \"400\","
+                                + " \"timestamp\": \"2022-06-03 16:18:12\"}")
+                  })
+            }),
+        @ApiResponse(
+            responseCode = "409",
+            description = "Email or phone number already in use",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"This email is already in use\"],"
+                                + " \"status\": \"400\","
+                                + " \"timestamp\": \"2022-06-03 16:18:12\"}")
+                  })
+            })
+      })
+  @io.swagger.v3.oas.annotations.parameters.RequestBody(
+      content =
+          @Content(
+              schema = @Schema(implementation = UserRegisterRequest.class),
+              mediaType = "application/json",
+              examples = {
+                @ExampleObject(
+                    value =
+                        "{\"email\": \"username@gmail.com\","
+                            + " \"password\": \"paSword1\","
+                            + "  \"name\": \"Name\","
+                            + "  \"surname\": \"Surname\","
+                            + "  \"phoneNumber\": \"38166575757\","
```

Comment:
Maybe add the phone format here if you have any ?

Reply:
When schema is opened in Request body, there is shown a pattern for phone number, which is only number. We didn't specify it any further